dataScripts/tessTrain/tessTrain.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+set -e
+
+# Tested on a bare install of Ubuntu 22.04.3 LTS - 02/27/2024
+# Author: Charles@CharlesBock.com - github.com/Penguin2600
```

Comment:
If you want to add author info here, (if I recall correctly), it should go right after the shebang. Do correct me if I'm wrong!

```suggestion
#!/bin/bash
# Author: Charles Bock
# Email: Charles@CharlesBock.com
# Date: 2024-02-27
# Tested on a bare install of Ubuntu 22.04.3 LTS

set -e
```

Reply:
I'm good for whatever you like, Just wanted to make sure if some poor sap runs into this in the future they know who they can contact for help.